lib/use_cases/administrator/health_checks/fetcher.rb

```diff
@@ -0,0 +1,39 @@
+module UseCases
+  module Administrator
+    module HealthChecks
+      class Fetcher
+        def initialize(route53_gateway: Gateways::Route53.new, ips:)
```

Comment:
I'd switch those 2 arguments around.

https://github.com/rubocop-hq/ruby-style-guide#optional-arguments

In case of kwargs, `ips:` is required and `route53_gateway:` is our optional argument.